### PR TITLE
Change display location of client logo

### PR DESCRIPTION
**Pull request by Astach**

### CODE REVIEW

File: example/src/main/res/drawable/ic_example_logo.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<vector android:height="24dp" android:viewportHeight="32" android:viewportWidth="32" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#C9CACC" android:fillType="evenOdd" android:pathData="M16 0C7.2 0 0 7.2 0 16s7.2 16 16 16 16-7.2 16-16S24.8 0 16 0zm4 29.7V25c0-1.5-0.5-2.5-1.1-3 3.6-0.4 7.3-1.8 7.3-7.9 0-1.7-0.6-3.2-1.6-4.3 0.2-0.4 0.7-2-0.2-4.2 0 0-1.3-0.4-4.4 1.6-1.3-0.4-2.6-0.5-4-0.5s-2.7 0.2-4 0.5c-3.1-2-4.4-1.6-4.4-1.6-0.9 2.2-0.3 3.9-0.2 4.3-1 1.1-1.6 2.5-1.6 4.3 0 6.1 3.7 7.5 7.3 7.9-0.5 0.4-0.9 1.1-1 2.1-0.9 0.4-3.2 1.1-4.7-1.3 0 0-0.8-1.5-2.5-1.6 0 0-1.6 0-0.1 1 0 0 1 0.5 1.8 2.3 0 0 0.9 3.1 5.4 2.1v2.9c-5.8-1.7-10-7-10-13.3 0-7.7 6.3-14 14-14s14 6.3 14 14c0 6.3-4.2 11.6-10 13.4z"/>
```
Comment (Nexcius) [inline example/src/main/res/drawable/ic_example_logo.xml]: `android:fillType="evenOdd"` is not supported on earlier devices. How does it look there?

Comment (Nexcius) [inline example/src/main/res/drawable/ic_example_logo.xml]: The path data is very long and should be simplified

File: example/lint.xml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <lint>
     <issue id="IconExpectedSize" severity="ignore"/>
+
+    <issue id="UnusedAttribute" severity="ignore" />
```
[the remark this answers is not in the document]
Comment (Astach) [reply example/lint.xml]: 1/ the attribute is used on recent devices api 24 + If I'm right.
2/ the path is already optimized using this tool https://github.com/alexjlockwood/avocado that reduced significantly the path and size but the lint issue still appears, I don't know any other way to optimize more the SVG.

Those lint issues don't make sense all the time (especially the first one)

File: example/src/main/res/drawable/ic_example_logo.xml (hunk 1 above)
Comment (Astach) [reply example/src/main/res/drawable/ic_example_logo.xml]: It looks the same test o API 19

Comment (Astach) [reply example/src/main/res/drawable/ic_example_logo.xml]: Already simplified

**Comment by Nexcius** [comment]
I don't think disabling the lint checks is a good idea. Please add them to the vector specifically with `tools:ignore="UnusedAttribute,VectorPath"`

**Pull request merged**